.travis.yml

```diff
@@ -24,6 +25,5 @@ notifications:
     on_failure: change
 env:
 - secure: KG+szfdd5qhWb5C/kz05BND8wkEreqSOlZOWc9o2XXH2wFoabNUbA90mK2WeSR99gzoPqeydphrGwco+UPfoXcAwowhk3PzVOAZPhMlEX3G2znpeq8d17k4cQ/KoqpO1upKiADYqmxHkMaxjxb8JDOmeCqhhW5HYcOvTNUHdH2I=
-after_success:
-  git log -n 1 --format=%B | grep -iq "covr"; if [ $? -eq 0 ]; then echo "Checking test coverage with covr"; Rscript -e 'library(covr);coveralls()'; fi
-
+after_success: git log -n 1 --format=%B | grep -iq "covr"; if [ $? -eq 0 ]; then echo
```

Comment:
But we won't know if it causes trouble unless that gets executed, i.e. when the commit message includes `[covr]`. This is conditional on the commit msg.